### DISCUSSION TITLE
Are quoted key names permitted (too)?

### DISCUSSION THREAD

**Discussion by beedell-roke** [json5/json5-spec#48] [Q&A] [answered]
Does JSON5 allow standard `{"Quoted Key Name": ""}` like JSON(C) does, or am I restricted to unquoted key names (`{QuotedKeyName: ""}`)?

**Comment by jordanbtucker** [comment, marked as answer]
Yes, JSON5 is a superset of JSON, which means that any valid JSON document can be parsed as JSON5.
